deployment/docker/client/1604/Dockerfile

```diff
@@ -19,4 +23,5 @@ RUN virtualenv /lnt/venv
 ADD . /lnt
 
 RUN . /lnt/venv/bin/activate
+RUN pip install -r /lnt/requirements.txt
```

Comment:
If we are delegating to requirements.txt the installation of dependancies, can we remove the other `pip install` commands above?

Reply:
fixed